packages/expo-yarn-workspaces/bin/symlink-necessary-packages.js

```diff
@@ -15,13 +15,34 @@ function symlinkNecessaryPackages(projectPath) {
   projectPath = path.resolve(projectPath);
 
   symlinkNecessaryPackage(projectPath, 'expo');
-  symlinkNecessaryPackage(projectPath, 'expo-camera');
-  symlinkNecessaryPackage(projectPath, 'jsc-android');
-  symlinkNecessaryPackage(projectPath, 'hermes-engine');
   symlinkNecessaryPackage(projectPath, 'react-native');
-  symlinkNecessaryPackage(projectPath, 'react-native-unimodules');
-  symlinkNecessaryPackage(projectPath, '@react-native-community/cli-platform-ios');
-  symlinkNecessaryPackage(projectPath, '@react-native-community/cli-platform-android');
+  // Required for bare workflow gradle and Podfile imports which don't have access to node module resolution.
+  symlinkNecessaryPackage(projectPath, "jsc-android");
+  symlinkNecessaryPackage(projectPath, "hermes-engine");
+  symlinkNecessaryPackage(projectPath, "react-native-unimodules");
+  symlinkNecessaryPackage(projectPath, "@react-native-community/cli-platform-ios");
+  symlinkNecessaryPackage(projectPath, "@react-native-community/cli-platform-android");
```

Comment:
I kept these modules in because other bare workflow expo projects will need them. expo-camera has been extracted because it's not required for bare projects to run.